app/Models/WinRMService.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
This file looks weird and unused.

Reply:
You are correct. It looks like the file got saved on accident. Thank you for pointing that out. I will work through the failures as i can.

Reply:
Got all checks passing. Apologize for so many commits, its been about 12 years since I touched PHP and my first go with Python. Nice being back to my roots tho. 

In the future is there a way to run the checks locally? 

./lnms dev:check did help with the initial issues but seems to show passed even when github checks fail.